pom.xml

```diff
@@ -66,18 +66,20 @@
         <no-test-jar>false</no-test-jar>
         <git-plugin.version>3.0.5</git-plugin.version>
         <workflow-scm-step-plugin.version>2.4</workflow-scm-step-plugin.version>
-        <workflow-step-api-plugin.version>2.10</workflow-step-api-plugin.version>
+        <workflow-step-api-plugin.version>2.13-deferactionwrite2-20170901.210414-1</workflow-step-api-plugin.version>
```

Comment:
Probably not required but I'm being a bit extra cautious to ensure these dependencies always are added together.  I don't want maven to be "creative."